Wireframe/index.html

```diff
@@ -10,23 +10,58 @@
     <header>
       <h1>Wireframe</h1>
       <p>
-        This is the default, provided code and no changes have been made yet.
+        Let's explore three  concepts.
       </p>
     </header>
     <main>
+
+
       <article>
         <img src="placeholder.svg" alt="" />
```

Comment:
Can you review the required attributes of the `<img>` element? You're missing an important one.